### PR TITLE
Styling labels and text inputs

### PR DESCRIPTION
**Pull request by zerline**
Adding `background-color`, `color`, `font-size` to labels and all text inputs.
Adding `font-family`,  `font-style`, `font-variant`, `text-color`, `text-decoration` to labels.

### CODE REVIEW

File: packages/controls/src/widget_string.ts [hunk 1]
```diff
@@ -21,6 +25,105 @@ import * as _ from 'underscore';
...
+            _model_name: 'TextStyleModel',
+            _model_module: '@jupyter-widgets/controls',
+            _model_module_version: JUPYTER_CONTROLS_VERSION,
+        });
+    }
+
+    public static styleProperties = {
+        ...DescriptionStyleModel.styleProperties,
+        background_color: {
+            selector: '.input',
```
Comment (jasongrout) [inline packages/controls/src/widget_string.ts]: I think this should just be `'input'`, not `'.input'`? In CSS, prefixing the word with a dot means to look for a CSS class `input`, whereas not having a dot means to look for a tag with name `input`.

Comment (zerline) [reply packages/controls/src/widget_string.ts]: yes, that was meant to be `input` as a CSS class (which I have also added in the HTML display).
We could also consider the tag. If that works (I have not tested that), and if we can promise to have such an input tag for each and every text input.

Comment (zerline) [reply packages/controls/src/widget_string.ts]: ... which is not the case with `textarea.`

**Comment by zerline** [comment]
See #2728 or #2730.

**Pull request closed without merging**